### PR TITLE
Update dss interfaces

### PR DESCRIPTION
**Pull request by iamchrissmith**
Add dss-interfaces and update dss-exec-lib so we can use `WardsAbstract` and `PsmAbstract` from there

### CODE REVIEW

**Review by naszam — APPROVED** [review]
LGTM

**Comment by iamchrissmith** [comment]
waiting until #33 is merged

**Review by brianmcmichael — APPROVED** [review]

**Pull request merged**
